.gitignore

```diff
@@ -17,4 +17,5 @@ coverage_html/
 presqt/osf/osf_api.json
 
 # Media Files directory
-mediafiles/
\ No newline at end of file
+mediafiles/downloads/
```

Comment:
We might need to make some adjustments to the Docker setup related to this. We can chat about it anytime soon. Not urgent.